plugin/pxr/maya/lib/usdMaya/userExportedAttributesUI.py

```diff
@@ -44,9 +44,9 @@
         from PySide6.QtGui import QStringListModel
 
 try:
-    from PySide2.QtGui import QAbstractItemView, QCheckBox, QComboBox, QLabel, QListView, QPushButton, QStyledItemDelegate, QTableView, QVBoxLayout, QWidget
+    from PySide2.QtWidgets import (QAbstractItemView, QCheckBox, QComboBox, QLabel, QListView, QPushButton, QStyledItemDelegate, QTableView, QVBoxLayout, QWidget)
 except:
-    from PySide6.QtGui import QAbstractItemView, QCheckBox, QComboBox, QLabel, QListView, QPushButton, QStyledItemDelegate, QTableView, QVBoxLayout, QWidget
+    from PySide6.QtWidgets import (QAbstractItemView, QCheckBox, QComboBox, QLabel, QListView, QPushButton, QStyledItemDelegate, QTableView, QVBoxLayout, QWidget)
```

Comment:
@dj-mcg Can you confirm: QVBoxLayout, QLabel, QStyledItemDelegate. I think these ones are in QtGui.

Reply:
Looks like they are in QtWidgets for both PySide2 and PySide6:
https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QVBoxLayout.html
https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QLabel.html
https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QStyledItemDelegate.html

See QBrush (in QtGui) for comparison:
https://doc.qt.io/qtforpython-5/PySide2/QtGui/QBrush.html